bin/install-wp-tests.sh

```diff
@@ -1,78 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Because WP-CLI's Behat framework is responsible for setting up WordPress, this setup script is no longer necessary.